### ISSUE TITLE
Login modal improvements

### ISSUE THREAD

**Issue by wickr** [OregonDigital/OD2#1338]
### Descriptive summary

Login modal has very limited clickable areas for the radio buttons.

### Expected behavior

- [x] Logos and 'Other Affiliation' text should all be clickable areas to select a radio button
- [x] When the lower panel appears, "Continue to Log In" could have 'UO' or 'OSU' inserted to confirm the correct one is selected
- [x] Space before 'Other Affiliation' text starts
- [x] Logo images need alt text

### Related work

#1148 

### Accessibility Concerns

Add any information here to indicate any known or suspected accessibility issues for this ticket

**Comment by straleyb** [comment]
QA:
 - [ ] Logos and text next to radio buttons should be clickable
 - [ ] Better spacing for Other Affiliation
 - [ ] Footer button should include institution
 - [ ] Alt text exists

**Comment by CGillen** [comment]
Fixed by #1842

**Comment by jechols** [comment]
Sending this back - the improvements are implemented per this ticket, but the request was slightly incorrect. The images do not need an `alt` attribute because the textual content within the label already acts as alternative text. Having an alt attribute results in unnecessary verbosity for these radio buttons, and currently doesn't describe the functionality of the image. e.g., "University of Oregon Logo" describes the image, but not its *purpose*, which is choosing to login as a UO affiliate.

These images should either have null alt attributes (`alt=""`) or a value that describes their purpose without duplicating the existing label.
